protocol/javatests/org/openyolo/protocol/CredentialTest.java

```diff
@@ -69,6 +62,11 @@ public void testBuilder_emptyIdentifier() {
     }
 
     @Test(expected = RequireViolation.class)
+    public void testBuilder_emptyAuthMethod() {
+        new Credential.Builder(EMAIL_ID, "", "https://www.example.com");
```

Comment:
It is part of the test description, do you feel this is really necessary in this case?